crates/lib3h/src/transport/websocket/actor.rs

```diff
@@ -132,6 +175,8 @@ impl GhostTransportWebsocket {
 
                     if let Ok(url) = maybe_bound_url {
                         self.bound_url = Some(url.clone());
+                        self.advertise()
```

Comment:
We need to change the test so that it shows that this advertise worked after a bind.

Reply:
done